roles/calico/tasks/main.yml

```diff
@@ -7,20 +7,20 @@
     etcd_ca_host: "{{ groups.oo_etcd_to_config.0 }}"
     etcd_cert_subdir: "openshift-calico-{{ openshift.common.hostname }}"
 
-- name: Assure the calico certs have been generated
+- name: Calico Node | Assure the calico certs have been generated
   stat:
     path: "{{ item }}"
   with_items:
   - "{{ calico_etcd_ca_cert_file }}"
   - "{{ calico_etcd_cert_file}}"
   - "{{ calico_etcd_key_file }}"
 
-- name: Configure Calico service unit file
+- name:  Calico Node | Configure Calico service unit file
```

Comment:
Please, remove one space before `Calico Node prefix`. The same for the remaining 9 occurrences.